lib/engine/game/g_1848/game.rb

```diff
@@ -487,6 +528,13 @@ def after_buy_company(player, company, _price)
             end
           end
         end
+
+        def lookup_boe_price(p)
+          @stock_market.market[BOE_ROW].size.times do |i|
```

Comment:
.each instead of .size.times

Reply:
i copied this method from somewhere else, using each is way easier, i wonder why it's not used everywhere for price lookups.

Reply:
I'm not sure how the .times snuck in. Usually Toby (the primary reviewer) is good at catching those.